addons/website_event/views/event_templates.xml

```diff
@@ -11,710 +11,775 @@
-                    <hr/>
-                    <i class="fa fa-clock-o"/>
-                    <span>To</span>
-                    <span t-field="event.with_context(tz=event.date_tz).date_end" t-options='{"hide_seconds":"True"}'/>
-                </div>
-            </div>
-            <div class="card bg-secondary my-3" t-if="event.organizer_id">
-                <h4 class="card-header">Organizer</h4>
-                <div class="card-body">
-                    <div t-field="event.organizer_id" class="text-secondary font-weight-bold"/>
-                    <div itemprop="location" t-field="event.organizer_id" t-options='{
-                        "widget": "contact",
-                        "fields": ["phone", "mobile", "email"]
-                    }'/>
-                </div>
-            </div>
-            <div class="card bg-secondary my-3">
-                <h4 class="card-header">Social Stream</h4>
-                <div class="card-body">
-                    <t t-call="website_mail.follow"><t t-set="object" t-value="event"/></t>
-                    <div t-if="event.twitter_hashtag" class="mt16">
-                        <p><strong>Participate on Twitter</strong></p>
-                        <p class="text-muted">
-                            Find out what people see and say about this event,
-                            and join the conversation.
-                        </p>
-                        <p>
-                            <strong>Use this tag:</strong>
-                            <a t-att-href="'http://twitter.com/search?q=%23'+event.twitter_hashtag" target="_blank" class="badge badge-primary">#<span t-field="event.twitter_hashtag"/></a>
-                        </p>
+                        <!-- Organizer -->
+                        <div t-if="event.organizer_id" class="o_wevent_sidebar_block">
+                            <h6 class="o_wevent_sidebar_title">Organizer</h6>
+                            <h4 t-field="event.organizer_id"/>
+                            <div itemprop="location" t-field="event.organizer_id" t-options="{'widget': 'contact', 'fields': ['phone', 'mobile', 'email']}"/>
+                        </div>
+                        <!-- Social -->
+                        <div class="o_wevent_sidebar_block">
+                            <h6 class="o_wevent_sidebar_title">SHARE</h6>
+                            <p class="mb-2">Find out what people see and say about this event, and join the conversation.</p>
+                            <p t-if="event.twitter_hashtag" class="font-weight-bold">Hashtag: <a t-att-href="'http://twitter.com/search?q=%23'+event.twitter_hashtag" target="_blank">#<span t-field="event.twitter_hashtag"/></a></p>
+                            <div class="o_wevent_sidebar_social s_share mx-n1">
+                                <!-- TODO - Fix Facebook & Linkedin URL -->
```

Comment:
@qsm-odoo It's based on the s_share snippet but I can't figure out why it's working here.

Reply:
I'll check

Reply:
It worked on my side except you did not put a target="_blank" but I refactored to use the s_share snippet.